OpenRA.Game/Traits/World/Shroud.cs

```diff
@@ -112,7 +108,7 @@ static IEnumerable<CPos> FindVisibleTiles(World world, CPos position, WRange rad
 			var limit = radius.RangeSquared;
 			var pos = map.CenterOfCell(position);
 
-			foreach (var cell in map.FindTilesInCircle(position, r))
+			foreach (var cell in map.FindTilesInCircle(position, r, true))
```

Comment:
I don't understand while this needs to include tiles outside the current map bounds.